Emby.AutoOrganize/Emby.AutoOrganize.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyVersion>5.0.0</AssemblyVersion>
     <FileVersion>5.0.0</FileVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
This can be made one line

Reply:
How would you do that? I tried this but then the project wouldn't load:

```
<PropertyGroup Condition=" '$(Configuration)' == 'Release' " TreatWarningsAsErrors="true"/>
```

> error  : The attribute "TreatWarningsAsErrors" in element <PropertyGroup> is unrecognized.

Reply:
```xml
    <TreatWarningsAsErrors Condition=" '$(Configuration)' == 'Release' ">true</TreatWarningsAsErrors>
```
And put it in the `PropertyGroup` above